### PR TITLE
127 invite users page

### PR DESCRIPTION
**Pull request by tanmaythole**
My Organizations List Page - UI design

Tasks:
- [x] Create a redux store
- [X] Setup routing
- [X] UI design
- [ ] API integration

Ticket:
https://trello.com/c/T95SGVpX/127-invite-users-page-in-onboarding-organization

Screenshots:
Screenshot from 2023-01-03 18-51-22

### CODE REVIEW

File: src/Components/OrganizationsComponents/InviteUsers/InviteMembersSelect.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+const InviteMembersSelect = ({ org, value, loadOptions, onChange }) => {
+  // helps to format label which created using AsyncCreatableSelect
+  const formatCreateLabel = (inputValue) => (
+    <div className="text-muted">
+      Invite <span className="text-dark">{inputValue}</span> to {org}
+    </div>
+  );
+
+  const isValidNewOption = (inputValue) => validateEmail(inputValue);
```
Comment (tanmaythole) [inline src/Components/OrganizationsComponents/InviteUsers/InviteMembersSelect.jsx:18]: Add comment

Comment (tanmaythole) [reply src/Components/OrganizationsComponents/InviteUsers/InviteMembersSelect.jsx:18]: done 👍🏻

File: src/Components/OrganizationsComponents/InviteUsers/InviteMembersSelect.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <div className="text-muted">
+      Invite <span className="text-dark">{inputValue}</span> to {org}
+    </div>
+  );
+
+  // check whether the new option is valid or not
+  const isValidNewOption = (inputValue) => validateEmail(inputValue);
+
+  return (
+    <CreatableAsyncSelect
```
Comment (ChinmaySonawane) [inline src/Components/OrganizationsComponents/InviteUsers/InviteMembersSelect.jsx:39]: In this case,
`isMulti`, `cacheOptions`, `defaultOptions `are optional so it should be placed after required props

Comment (tanmaythole) [reply src/Components/OrganizationsComponents/InviteUsers/InviteMembersSelect.jsx:39]: done 👍🏻

**Review by ChinmaySonawane — APPROVED** [review]

**Pull request merged**